mmf/configs/datasets/localized_narratives/defaults.yaml

```diff
@@ -0,0 +1,49 @@
+dataset_config:
```

Comment:
As a rule of thumb, defaults should contain a sensible configuration of the model in its native format. Your work is currently experimental and should go into a different configuration variant under the projects folder where you override the defaults and add your own processors and configuration attributes there.